db/migrate/20200615130642_add_bought_cinstances_count_to_accounts.rb

```diff
@@ -0,0 +1,22 @@
+class AddBoughtCinstancesCountToAccounts < ActiveRecord::Migration[5.0]
```

Comment:
This needs to be migrated for the 3 schemas 😝

Reply:
And please ensure that it works for the 3 DBs also when it contains data for the affected tables/columns